libra_swarm/src/swarm.rs

```diff
@@ -273,7 +273,7 @@ impl LibraSwarm {
                 Ok(swarm) => {
                     return swarm;
                 }
-                Err(e) => error!("Error launching swarm: {}", e),
+                Err(e) => panic!("Error launching swarm: {}", e),
```

Comment:
This seems unrelated.

Reply:
Good call, my machine had some transient issue I was trying to debug related to writes to disk. Cleaning up now...